## Motor driver configuration for ETQuad

# Control Modes
# 1: Torque control
# 2: Velocity control
# 3: Position control

motor0:
  id: 0
  serial_number: "335837603230" # USB only
  control_mode: 3
  gear_ratio: 6.0
  # zero position
  # can id

motor1:
  id: 1
  serial_number: "3355375E3230"
  control_mode: 3
  gear_ratio: 6.0

motor2:
  id: 2
  serial_number: "--"
  control_mode: 3
  gear_ratio: 6.0

motor3:
  id: 3
  serial_number: "--"
  control_mode: 3
  gear_ratio: 6.0

motor4:
  id: 4
  serial_number: "--"
  control_mode: 3
  gear_ratio: 6.0

motor5:
  id: 5
  serial_number: "--"
  control_mode: 3
  gear_ratio: 6.0

motor6:
  id: 6
  serial_number: "--"
  control_mode: 3
  gear_ratio: 6.0

motor7:
  id: 7
  serial_number: "--"
  control_mode: 3
  gear_ratio: 6.0
